### PR TITLE
Updated Signup form to make gender selections more inclusive

### PR DESCRIPTION
**Pull request by JGibbsWork**
Changed value and verbiage for gender selection on signup page. Tested to see if the new value was consistent on individual profile page after sign up.

**Backend checklist**
- [ ] Formatted my code by running `autoflake -r -i --remove-all-unused-imports src && isort . && black .` in `app/backend`
- [ ] Added tests for any new code or added a regression test if fixing a bug
- [ ] All tests pass
- [ ] Run the backend locally and it works
- [ ] Added migrations if there are any database changes, rebased onto `develop` if necessary for linear migration history

**Frontend checklist**
- [X] Formatted my code with `yarn format && yarn lint --fix`
- [X] There are no warnings from `yarn lint`
- [X] There are no console warnings when running the app
- [X] Added any new components to storybook
- [X] Added tests where relevant
- [X] All tests pass
- [X] Clicked around my changes running locally and it works
- [X] Checked Desktop, Mobile and Tablet screen sizes

### CODE REVIEW

File: app/frontend/src/features/auth/constants.tsx [hunk 1]
```diff
@@ -33,7 +33,7 @@ export const CONTACT =
   "Admins will be happy to change your gender label. Please contact support@couchers.org!";
 export const CURRENT_PASSWORD = "Current password";
 export const ENTER_EMAIL = "Enter your username/email";
-export const FEMALE = "Female";
+export const WOMAN = "A Woman";
```
Comment (lucaslcode) [inline app/frontend/src/features/auth/constants.tsx]: Can this be A_WOMAN so we don't accidentally use it in the wrong place :D

**Review by lucaslcode — CHANGES_REQUESTED** [review]

File: app/frontend/src/features/auth/constants.tsx (hunk 1 above)
Comment (JGibbsWork) [reply app/frontend/src/features/auth/constants.tsx]: Sounds good! I'll give it an update!

**Review by JGibbsWork — COMMENTED** [review]
Fixed it!

**Comment by aapeliv** [comment]
@JGibbsWork: we changed the signup form, so I think this PR might need some major changed, sorry!

Also, please re-request review, might be that Lucas doesn't notice otherwise!

**Review by lucaslcode — DISMISSED** [review]
This looks good but needs merge conflicts solved (sorry for being slow)

**Comment by JGibbsWork** [comment]
No worries! I've been a bit slow on my end too. I'll get this sorted today.

**Comment by lucaslcode** [comment]
This was partly fixed in #1641 (remember to self-assign issues!) but the grammar is more correct here so fix the merge conflicts and we'll merge this :)

**Comment by JGibbsWork** [comment]
I think this should fix it, did a rebase and needed to do a force push. Please let me know if there are any issues here and I am happy to make some changes!

**Comment by lucaslcode** [comment]
Failing tests due to renamed constants

**Comment by Vinalti** [comment]
Maybe it would be better to keep the constant name as `WOMAN` / `MAN` instead of `A_WOMAN` `A_MAN` (so if the constant is used somewhere else it doesn't break it) and just change it's value. What do you think @JGibbsWork ? 

```diff
- export const WOMAN = "Woman";
+ export const A_WOMAN = "A Woman";
  export const FULL_NAME = "Full name";
  export const GENDER_LABEL = "I identify as ...";
  export const GENDER_REQUIRED = "Please select your gender identity";
@@ -49,7 +49,7 @@ 
  export const LOGIN = "Log in";
  export const LOGIN_HEADER = "Welcome back!";
  export const LOGIN_PAGE = "Log in page";
- export const MAN = "Man";
+ export const A_MAN = "A Man";
```

**Comment by lucaslcode** [comment]
Closing this as it's become stale. Feel free to re-open.

**Pull request closed without merging**